.gitignore

```diff
@@ -3,6 +3,7 @@
 .rspec
 /test/dummy/log/*.log
 /test/dummy/tmp
+/coverage/*
```

Comment:
This isn't right: we want the `.last_run.json`. When we *don't* skip CI, we can use that file to enable a feature where PRs that reduce coverage will fail.

In general, though, we like to have that file so that humans can get a quick sense of the coverage as well.